src/index.html

```diff
@@ -5,10 +5,38 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header" id="home">
+      <div class="header__logo">
+        <a href="./index.html" class="logo__link">
+          <img src="./images/logo.png" alt="MOYO logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="nav__links is-active" href="#">Apple</a></li>
```

Comment:
Apply this kind of indentation everywhere
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__links is-active">Apple</a>
          </li>
```